sbt-plugin/src/sbt-test/testing/multi-framework/project/build.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % sys.props("plugin.version"))
```

Comment:
Here you've inlined `sys.props(...)` but in other tests you used a separate `val scalaJSVersion`. Consider making this uniform, one way or the other.